src/impression.js

```diff
@@ -187,16 +190,26 @@ function invoke(win, clickUrl) {
     credentials: 'include',
```

Comment:
Wondering if it's appropriate to continue to use "fetchJson" here?

Reply:
Hi

Reply:
I think `#fetchJson` is ok to use here. 
`#fetchJson` merely sets up the request to accept JSON. For the special no-content case, we resolve with null response with status 204 anyway. The real `response.json()` is not performed.

Reply:
Sounds good, thanks!